src/aoapplication.cpp

```diff
@@ -177,3 +177,71 @@ void AOApplication::call_announce_menu(Courtroom *court)
   AOCaseAnnouncerDialog announcer(nullptr, this, court);
   announcer.exec();
 }
+
+#ifdef BASSAUDIO
+// Callback for when BASS device is lost
+void CALLBACK AOApplication::BASSreset(HSTREAM handle, DWORD channel,
+                                       DWORD data, void *user)
+{
+  doBASSreset();
```

Comment:
So, this will always kill the music when the headphones are unplugged. I don't agree with this, but you told me that if I wanted to do that "big IQ" stuff I could do it myself. Okay.